logs/logWrapper.go

```diff
@@ -0,0 +1,31 @@
+package logs
+
+import (
+	"io"
+)
+
+type stdoutWriter struct {
+	writer io.Writer
+}
+
+type stderrorWriter struct {
+	writer io.Writer
+}
+type stdbothWriter struct {
```

Comment:
Was going to use it for the tty case for got to ask you if you would prefer that?
That way you can tell you dont have differentiated logs.